packaging/msi/make.ps1

```diff
@@ -0,0 +1,67 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+<#
+.SYNOPSIS
+    Makefile like build commands for the Collector on Windows.
+    
+    Usage:   .\make.ps1 <Command> [-<Param> <Value> ...]
+    Example: .\make.ps1 New-MSI -Config "./my-config.yaml"
+.PARAMETER Target
+    Build target to run (Install-Tools, New-MSI)
+#>
+Param(
+    [Parameter(Mandatory=$true, ValueFromRemainingArguments=$true)][string]$Target
+)
+
+$ErrorActionPreference = "Stop"
+
+function Install-Tools {
+    choco install wixtoolset -y
+    setx /m PATH "%PATH%;C:\Program Files (x86)\WiX Toolset v3.11\bin"
+    refreshenv
+}
+
+function New-MSI(
+    [string]$Config="./examples/otel-local-config.yaml"
+) {
+    $Version = If($env:VERSION) { $env:VERSION } else { "0.0.1" }
```

Comment:
We should also set `VERSION` to the `CIRCLE_TAG` env var (if it exists, without the `v`) for release builds.

Reply:
Done - since we're referring to a CircleCI specific env variable, I moved this code into the CircleCI config file.

Minor note: MSI [Product Version](https://docs.microsoft.com/en-us/windows/win32/msi/productversion) only supports the regex format you specified (e.g. 0.0.1). The tag filter we use to determine when to publish releases allows additional characters at the end (e.g. v0.0.1-alpha). If we see a tag like that I've opted to just ignore the extra chars.